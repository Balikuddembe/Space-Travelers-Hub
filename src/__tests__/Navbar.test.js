import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../Redux/ConfigureStore';
import NavBar from '../components/Navbar/Navbar';

describe('Navbar componente test', () => {
  test('should render', () => {
    const navBar = render(
      <Provider store={store}>
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
        ,
      </Provider>,
    );
    expect(navBar).toMatchSnapshot();
  });
});
