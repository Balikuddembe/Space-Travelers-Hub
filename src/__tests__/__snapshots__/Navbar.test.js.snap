// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar componente test should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="NavBar"
      >
        <div
          class="LogoContainer"
        >
          <img
            alt=""
            class="planet"
            src="planet.png"
          />
          <h1>
            Space Traverlers Hub
          </h1>
        </div>
        <div
          class="NavLinks"
        >
          <a
            href="/Rockets.js"
          >
            Rockets
          </a>
          <a
            href="/Missions.js"
          >
            Missions
          </a>
          <a
            aria-current="page"
            class="currentPage"
            href="/"
          >
            My Profile
          </a>
        </div>
      </div>
      ,
    </div>
  </body>,
  "container": <div>
    <div
      class="NavBar"
    >
      <div
        class="LogoContainer"
      >
        <img
          alt=""
          class="planet"
          src="planet.png"
        />
        <h1>
          Space Traverlers Hub
        </h1>
      </div>
      <div
        class="NavLinks"
      >
        <a
          href="/Rockets.js"
        >
          Rockets
        </a>
        <a
          href="/Missions.js"
        >
          Missions
        </a>
        <a
          aria-current="page"
          class="currentPage"
          href="/"
        >
          My Profile
        </a>
      </div>
    </div>
    ,
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
